lib/service/account_service.dart

```diff
@@ -658,29 +648,48 @@ class AccountServiceImpl extends AccountService {
         'isDoneOnboarding: $isDoneOnboarding, '
         'defaultWallet: ${defaultWallet?.uuid}');
 
-    /// in previous version, we assume that if user has primary address,
-    /// that is their default account (using to save cloud data base)
-    /// but if user has no primary address, but have backup version,
-    /// we will take the first uuid as default account
+    /// Migrate passkeys note:
+    /// To simplify this migration on mobile, we will not support restoring
+    /// backed-up data for users who uninstall and then reinstall apps that
+    /// still use didKey and the Tezos primary address.
+    /// The data that will not be restored includes playlists, settings,
+    /// and derived addresses. However, we will automatically derive a pair
+    /// of addresses for each wallet that users own.
     // case 1: complete new user, no primary address, no backup keychain
     // nothing to do other than create new wallet
+    /// create passkeys, no need to migrate
     if (defaultWallet == null) {
       log.info('[AccountService] migrateAccount: case 1 complete new user');
-      await createNewWallet();
+      final wallet = await createNewWallet();
+      await _addressService.registerPrimaryAddress(
+          info: primary_address_channel.AddressInfo(
+        uuid: wallet.uuid,
+        chain: 'ethereum',
+        index: 0,
+      ));
+      await createLoginJwt();
+      await insertNextAddressFromUuid(wallet.uuid, WalletType.MultiChain,
+          name: '');
+      await androidBackupKeys();
```

Comment:
why we have to call thhis?

Reply:
take it out from `createNewWallet`
i don't qua to pass `createLoginJwt` into `createNewWallet`